.github/workflows/test-prod.yml

```diff
@@ -27,8 +27,8 @@ jobs:
         run: go test -parallel 20 -coverprofile=coverage.txt -coverpkg=./... ./...
         env:
           SPACELIFT_API_KEY_ENDPOINT: ${{ secrets.PROD_SPACELIFT_API_KEY_ENDPOINT }}
-          SPACELIFT_API_KEY_ID: ${{ secrets.PROD_SPACELIFT_API_KEY_ID }}
-          SPACELIFT_API_KEY_SECRET: ${{ secrets.PROD_SPACELIFT_API_KEY_SECRET }}
+          SPACELIFT_API_KEY_ID: ${{ secrets.PROD_SPACELIFT_TEST_API_KEY_ID }}
+          SPACELIFT_API_KEY_SECRET: ${{ secrets.PROD_SPACELIFT_TEST_API_KEY_SECRET }}
```

Comment:
machine user?

Reply:
yeah, I'm going to clean up the secrets of this repository one day.
`PROD_SPACELIFT_TEST_API_KEY_ID`, `PROD_SPACELIFT_RELEASE_API_KEY_ID`, ...